python/docs/source/reference/pyspark.sql.rst

```diff
@@ -603,3 +603,30 @@ Grouping
     GroupedData.pivot
     GroupedData.sum
     PandasCogroupedOps.applyInPandas
+
+Catalog APIs
```

Comment:
Thanks, @dominikgehl. Can you also add `Catalog` class to the `Core Classes` above?

Reply:
Done